sdk/iot/hub/src/az_iot_hub_client_twin.c

```diff
@@ -112,7 +112,7 @@ AZ_NODISCARD az_result az_iot_hub_client_twin_document_get_publish_topic(
       + az_span_size(az_iot_hub_twin_get_pub_topic)
       + (int32_t)sizeof(az_iot_hub_client_twin_question)
       + az_span_size(az_iot_hub_client_request_id_span)
-      + (int32_t)sizeof(az_iot_hub_client_twin_equals) + az_span_size(request_id);
+      + (int32_t)sizeof(az_iot_hub_client_twin_equals) + _az_iot_u32toa_size(request_id);
```

Comment:
move this to the next line, like the previous

Reply:
The `.clang-format` will put it back if you run it

Reply:
if you even need to use manual formatting and you need to ask clang-format to ignore it, you can use 
`// clang-format off`  and `// clang-format on` like
```c
// clang-format off
you code that won't be auto - formatted
// clang-format on
```